salt/modules/zypper.py

```diff
@@ -649,10 +649,10 @@ def version_cmp(ver1, ver2, ignore_epoch=False):
     return __salt__['lowpkg.version_cmp'](ver1, ver2, ignore_epoch=ignore_epoch)
 
 
-def list_pkgs(versions_as_list=False, **kwargs):
+def list_pkgs(versions_as_list=False, attr=None, **kwargs):
```

Comment:
> I suggest you supply all information by default.

I disagree here.

Reason is that the returned data has a different structure when the `attr` parameter is set, so any third party application or script must be ready to accept it.

That means that putting the `attr` parameter as a default will make `zypper` return something different from all other backends that must be treated in a special way by default, and will break backwards compatibility. This is not really the intention here - what I would like to see is an option to return more data only if the requester needs it, only if it is prepared to handle it.